examples/multisensor/README.md

```
@@ -0,0 +1,89 @@
+> target device.
+
+```bash
+pip install tartanair
+python3 download_tartan.py
+```
+
+> **Troubleshooting**: If the download fails (for example, with a connection
+> timeout to `airlab-share-02.andrew.cmu.edu`), the PyPI version may be
+> outdated. Install the latest version directly from GitHub (you may also
+> need to fix tartanair API calls in the download script):
+> ```bash
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)